providers/flagd/README.md

```diff
@@ -51,6 +51,20 @@ FlagdProvider flagdProvider = new FlagdProvider(
 
 In the above example, in-process handlers attempt to connect to a sync service on address `localhost:8013` to obtain [flag definitions](https://github.com/open-feature/schemas/blob/main/json/flagd-definitions.json).
 
+In-process resolver can also work in an offline mode. To enable this mode, you should provide a valid flag configuration file with the option `offlineFlagSourcePath`.
+The file must contain a valid flagd flag source file.
+
+```java
+FlagdProvider flagdProvider = new FlagdProvider(
+        FlagdOptions.builder()
+                .resolverType(Config.Evaluator.IN_PROCESS)
+                .offlineFlagSourcePath("PATH")
```

Comment:
Initially I though calling this "file mode" or something was better, but I think since we haven't implemented a watcher or anything, it's probably better to call it "offline mode" for now, since it's really not a feature-full replacement for the gRPC sync.

Reply:
Yup, this is purely a development time feature (tests, local validations).